src/helm/benchmark/window_services/wider_openai_window_service.py

```diff
@@ -10,7 +10,13 @@ def max_sequence_length(self) -> int:
         return 4000
 
 
-class GPT4WindowService(GPT2WindowService):
+class OpenAIChatWindowService(WiderOpenAIWindowService):
+    @property
+    def tokenizer_name(self) -> str:
+        return "openai/cl100k_base"
+
+
+class GPT4WindowService(OpenAIChatWindowService):
```

Comment:
Could you check if the special tokens are the same as in GPT2, or if we need to update those as well?

Reply:
I found this: `"<|fim_prefix|>"` which seems to be different.

Reply:
Hmm that seems to be for infilling. Let's default to the same token as GPT-2.